src/scene/constants.js

```diff
@@ -626,6 +626,14 @@ export const TONEMAP_ACES2 = 4;
  */
 export const TONEMAP_NEUTRAL = 5;
 
+/**
+ * No tonemapping nor the exposure is applied. Used for HDR rendering.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const TONEMAP_NONE = 6;
```

Comment:
How dangerous would it be to renumber these constants so that `TONEMAP_NONE` was 0? I mean, no one is setting via numbers in code. But presumably numbers are stored in the database...

Reply:
yeah the Editor projects give us numbers .. and so I would prefer to not break that.